### PR TITLE
dovecot: update to 2.3.6

### PR DESCRIPTION
**Pull request by lucize**
Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

Maintainer: me 
Tested: x86

### CODE REVIEW

File: mail/dovecot/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 PKG_NAME:=dovecot
-PKG_VERSION:=2.3.5.2
+PKG_VERSION:=2.3.6
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.dovecot.org/releases/2.3
-PKG_HASH:=ba14e41aefd81a868a35b83bcb54194116106424d37690519b50ea83c0f31bf2
+PKG_HASH:=ed1d8dc1beeae9c6c73deac73a62ef19fe9262fbffd86604a3f690452f5536c7
 PKG_LICENSE:=LGPL-2.1 MIT BSD-3-Clause Unique
```
Comment (diizzyy) [inline mail/dovecot/Makefile]: Can you update the LGPL tag?
https://spdx.org/licenses/
What does Unique refer to?

**Comment by diizzyy** [comment]
Add ```--without-docs```?
Looks good otherwise

**Comment by lucize** [comment]
--without-docs the examples will not be packed, I'll add that

File: mail/dovecot/Makefile (hunk 1 above)
Comment (lucize) [reply mail/dovecot/Makefile]: is has LGPL and MIT, I don't know either what is with the unique stuff

Comment (lucize) [reply mail/dovecot/Makefile]: all stuff is at the beginning of https://github.com/dovecot/core/blob/master/COPYING

Comment (diizzyy) [reply mail/dovecot/Makefile]: The LGPL "tag" isn't valid, please see the link I provided.
Drop the unique tag
It's not complete by better than before after those changes

**Comment by lucize** [comment]
@diizzyy done

**Comment by diizzyy** [comment]
LGTM

**Pull request merged**